data/assets/scene/solarsystem/missions/osirisrex/bennu.asset

```diff
@@ -16,39 +25,20 @@ local Bennu = {
     },
     Scale = {
       Type = "StaticScale",
-      Scale = 1000.0
+      Scale = 1.0
```

Comment:
If the scale is set to 1.0 then couldn’t it just be removed to make the code cleaner?

Reply:
Agreed, since I no longer needed to scale the model.